backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1018,28 +1018,17 @@ databaseChangeLog:
                   name: ordering_provider_id
                   type: uuid
                   constraints:
-                    nullable: false
+                  #   nullable: false
```

Comment:
this one worries me, because currently its not nullable in our db, do you think it is made not null in a later migration

Reply:
honestly, I dont think we'll ever need to run this rollback, and if we do having it null-able will not break anything (from migration perspective) since its less constraining.

Reply:
This column doesn't exist in `organization` any more - it gets dropped in this change set. Prior to being dropped, though, it was non-nullable. I think I see what to do here, though - I think `clia_number` and `ordering_provider_id` can be copied from `facility`.

As far as the error goes, it's the same error as with `clia_number` - if you make this non-nullable, then you need to provide default values for `ordering_provider_id`.

Reply:
I just realized that `clia_number` and `ordering_provider_id` in `facility` are also nullable so that doesn't solve the problem either. I'll still use them as the source of the new column values, but I'll leave the new columns nullable.